Index.html

```diff
@@ -244,33 +244,35 @@ <h4>
         </section>
       </li>
       <li>
-        <section id="awards_&_interest">
-          <div class="awards">
+        <section id="awardsInterest">
+          <div class="awardsInterest__list awards">
             <h3>
               Awards
             </h3>
             <p>
-              Lorem ipsum dolor sit amet, consectetur adipisicing elit. 
+              Where I shine 
             </p>
-            <ul>
-              <li>Morbi in sem quis dui placerat ornare. </li>
-              <li>Praesent dapibus, neque id cursus faucibu.</li>
-              <li>Phasellus ultrices nulla quis nibh. Quisque a lectus.</li>
-              <li>Pellentesque fermentum dolor. Aliquam quam lectus.</li>
-            </ul>
+              <ul>
+                <li>Excellency Excellence in Mathematics.</li>
+                <li>Excellency Excellence in Natural Sciences.</li>
+                <li>Certificate of Merit in Spanish Language.</li>
+                <li>Drawing and Photography.</li>
+                <li>Literary Juvenile Literary Contest (PropaGas Foundation).</li>
+                <li>Gold, Silver and Bronze medals on Swimming.</li>
+              </ul>
           </div>
-          <div class="interest">
+          <div class="awardsInterest__list interest">
             <h3>
               Interest
             </h3>
             <p>
-              Lorem ipsum dolor sit amet, consectetur adipisicing elit. 
+              Out of office. 
             </p>
             <ul>
-              <li>Morbi in sem quis dui placerat ornare. </li>
-              <li>Praesent dapibus, neque id cursus faucibu.</li>
-              <li>Phasellus ultrices nulla quis nibh. Quisque a lectus.</li>
-              <li>Pellentesque fermentum dolor. Aliquam quam lectus.</li>
+              <li>Reading. </li>
+              <li>Drawing.</li>
+              <li>Cooking.</li>
+              <li>Photography.</li>
```

Comment:
ohhh I didn't know you like photography

Reply:
Nothing professional, just as a hobby. I helped my second uncle that used to have a studio here when I was 10 or 12 years old